tools/lint/lint.xml

```diff
@@ -114,4 +115,11 @@
 
     <!-- DI -->
     <issue id="JvmStaticProvidesInObjectDetector" severity="error" />
+
+    <!-- Temporary ignoring errors... TODO Unignore at some point -->
+    <issue id="LintError" severity="error">
+        <ignore path="**/CreateRoomViewModel.kt" />
+        <ignore path="**/MessageComposerViewModel.kt" />
+    </issue>
```

Comment:
Adding the issues in the baseline file does not work, so I added the exclusion here...